web/app/components/root/root.jsx

```diff
@@ -268,13 +268,13 @@ export default class Root extends Component {
 
           {
             isBlockedVisible && (
-              <div className="root__main">
+              <div className="root__main" role="region" aria-label="Blocked Users">
```

Comment:
`Blocked users`.

And may be it's better to move all this things to BlockedUsers component root node.